Patch-Source: https://sources.debian.org/src/libde265/1.0.16-1/debian/patches/fix-private-libs-pc.patch
--
Description: Fix missing "LIBS_PRIVATE" variable for libde265.pc in
 autoconf builds.
From 56d1cf52e43d734692721c99b72815c0fee1f184 Mon Sep 17 00:00:00 2001
From: Joachim Bauch <bauch@struktur.de>
Date: Thu, 21 Dec 2023 09:22:58 +0100
Subject: [PATCH] Fix missing "LIBS_PRIVATE" for libde265.pc in autoconf
 builds.
Forwarded: https://github.com/strukturag/libde265/pull/439
---
 configure.ac | 3 +++
 1 file changed, 3 insertions(+)

Index: libde265/configure.ac
===================================================================
--- libde265.orig/configure.ac
+++ libde265/configure.ac
@@ -105,6 +105,9 @@ AC_FUNC_MKTIME
 LIBS_PRIVATE="$LIBS -lstdc++"
 AC_SUBST(LIBS_PRIVATE)
 
+LIBS_PRIVATE="$LIBS -lstdc++"
+AC_SUBST(LIBS_PRIVATE)
+
 AM_CONDITIONAL(MINGW, expr $host : '.*-mingw' >/dev/null 2>&1)
 
 # Check if "__STRICT_ANSI__" is required.
